packages/core/test/fixtures/id.ts

```diff
@@ -42,28 +42,6 @@ export const machine = createMachine({
           id: 'B.dot'
         }
       }
-    },
-    getter: {
```

Comment:
was the support for getters dropped anywhere in this PR?

Reply:
It was dropped in a previous PR. Forgot to remove this.